_procurements/early-childhood-education-research-study.markdown

```diff
@@ -0,0 +1,8 @@
+---
+position: 0
+Full Title: Early Childhood Education Research Study
+Description: 'The 3.5 year USAID Education Data activity implemented by DevTech Systems, Inc. (DevTech) began in April 2018 with the objective to provide education assessment, data management, and research and evaluation services in support of the USAID Let’s Read Project that targets the Southern, Eastern, Muchinga, North Western, and West provinces of Zambia. Under USAID Education Data activity, DevTech is running a competitive procurement to hire a local survey firm to support data collection for its Early Childhood Education Research Study.'
```

Comment:
Are the single quotes needed here?  Best to leave them out in case you have `Bob's burgers` embedded somewhere in the text so you don't have to worry about interpolation.

Reply:
Pretty sure its needed with longer strings, let me test it out.

Reply:
Nice it works with out the single quotes!